public/index.html

```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="utf-8">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link
```

Comment:
We don't get this from the npm package somehow? Like, this is fine, but I'm surprised this is the way we have to do it

Reply:
Turns out your intuition is correct: https://react.semantic-ui.com/usage

Added the CSS import.